raiden/tests/fixtures/blockchain.py

```diff
@@ -96,7 +86,7 @@ def dagpath():
     return os.path.expanduser('~/.ethash/full-R23-0000000000000000')
 
 
-@pytest.fixture(scope='session', autouse=True)
+@pytest.fixture(autouse=True)
```

Comment:
blockchain_type is not a session fixture, so pregenerate_dag shouldn't be one, I don't understand how that was not failing before. This change has no loss of behavior.